plugins/available/base.plugin.bash

```diff
@@ -80,8 +80,8 @@ function mkcd ()
     example '$ mkcd foo'
     example '$ mkcd /tmp/img/photos/large'
     group 'base'
-    mkdir -p "$*"
-    cd "$*"
+    mkdir -p -- "$*"
+    cd -- "$*"
```

Comment:
Thanks! I hope no one really considers adding directories starting with a dash, but I'm glad we support it now!